ci-operator/config/openshift/knative-eventing-contrib/openshift-knative-eventing-contrib-release-v0.17.1__46.yaml

```diff
@@ -0,0 +1,158 @@
+  to: knative-eventing-sources-kafka-channel-dispatcher
+- dockerfile_path: openshift/ci-operator/knative-images/camel-source-controller/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/camel-source-controller
+  to: knative-eventing-sources-camel-source-controller
+- dockerfile_path: openshift/ci-operator/knative-images/kafka-source-controller/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/kafka-source-controller
+  to: knative-eventing-sources-kafka-source-controller
+- dockerfile_path: openshift/ci-operator/knative-images/event_display/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/event_display
+  to: knative-eventing-sources-event-display
+- dockerfile_path: openshift/ci-operator/knative-images/websocketsource/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/websocketsource
+  to: knative-eventing-sources-websocketsource
+- dockerfile_path: openshift/ci-operator/knative-images/appender/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/appender
+  to: knative-eventing-sources-appender
+- dockerfile_path: openshift/ci-operator/knative-images/kafka-source-adapter/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/kafka-source-adapter
+  to: knative-eventing-sources-kafka-source-adapter
+- dockerfile_path: openshift/ci-operator/knative-images/heartbeats/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/heartbeats
+  to: knative-eventing-sources-heartbeats
+- dockerfile_path: openshift/ci-operator/knative-images/kafka-channel-webhook/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/kafka-channel-webhook
+  to: knative-eventing-sources-kafka-channel-webhook
+- dockerfile_path: openshift/ci-operator/knative-images/github-source-controller/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/github-source-controller
+  to: knative-eventing-sources-github-source-controller
+- dockerfile_path: openshift/ci-operator/knative-test-images/kafka_performance/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/kafka_performance
+  to: knative-eventing-sources-test-kafka-performance
+- dockerfile_path: openshift/ci-operator/knative-test-images/kafka-publisher/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/kafka-publisher
+  to: knative-eventing-sources-test-kafka-publisher
+promotion:
```

Comment:
I believe this `promotion` tag is redundant because we already promote from the `45` variant. Here we basically promote to the same coordinates, overwriting what we did for `45`. We using this approach in Knative Serving where we only promote from the main variant.

Reply:
it's set to disabled, thought

Reply:
I'd rather keep the DISABLED, so if we remove one platform we, just have to toggle the flag, instead of forgetting to set it all ? 

I mean... it's autogenerated etc

Reply:
Oh. OK it's disabled. Fine.